sys/walltime/fallback.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2025 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     sys_walltime
+ * @{
+ *
+ * @file
+ * @brief       Wall-Clock ztimer fallback
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ *
+ * @}
+ */
+
+#include "rtc_utils.h"
+#include "walltime.h"
+#include "ztimer.h"
+
+static uint32_t ztimer_offset;
+
+__attribute__((weak))
+int walltime_impl_get(struct tm *time, uint16_t *ms)
+{
+    uint32_t now = ztimer_now(ZTIMER_MSEC);
```

Comment:
could benefit from #20494

Reply:
The walltime module is good from my perspective. As the xtimer_compat was merged, it would be nice to have it here.

Reply:
but tbh I'm not sure how useful a non-msec based backend would be here.
ZTIMER_USEC wraps after 1h and 11 minutes.

Reply:
If there is no implication of using `ztimer_msec` towards not using `ztimer_msec` then yes msec is better than usec here.
Then `ztimer_msec` should be added to the Makefile.dep.